src/node_report.cc

```diff
@@ -712,36 +803,39 @@ static void PrintStackFromStackTrace(FILE* fp, Isolate* isolate, DumpEvent event
  * Function to print a JavaScript stack frame from a V8 StackFrame object
  *
  ******************************************************************************/
-static void PrintStackFrame(FILE* fp, Isolate* isolate, Local<StackFrame> frame, int i, void* pc) {
+static void PrintStackFrame(std::ostream& out, Isolate* isolate, Local<StackFrame> frame, int i, void* pc) {
   Nan::Utf8String fn_name_s(frame->GetFunctionName());
   Nan::Utf8String script_name(frame->GetScriptName());
   const int line_number = frame->GetLineNumber();
   const int column = frame->GetColumn();
+  char buf[64];
 
   // First print the frame index and the instruction address
 #ifdef _WIN32
-  fprintf(fp, "%2d: [pc=0x%p] ", i, pc);
+  snprintf( buf, sizeof(buf), "%2d: [pc=0x%p] ", i, pc);
+  out << buf;
 #else
-  fprintf(fp, "%2d: [pc=%p] ", i, pc);
+  snprintf( buf, sizeof(buf), "%2d: [pc=%p] ", i, pc);
+  out << buf;
 #endif
 
   // Now print the JavaScript function name and source information
   if (frame->IsEval()) {
     if (frame->GetScriptId() == Message::kNoScriptIdInfo) {
-      fprintf(fp, "at [eval]:%i:%i\n", line_number, column);
+      out << "at [eval]:" << line_number << ":" << column << "\n";
     } else {
-      fprintf(fp, "at [eval] (%s:%i:%i)\n", *script_name, line_number, column);
+      out << "at [eval] (" << *script_name << ":" << line_number << ":" << column << ")\n";
     }
     return;
   }
 
   if (fn_name_s.length() == 0) {
-    fprintf(fp, "%s:%i:%i\n", *script_name, line_number, column);
+    out  << *script_name << ":" << line_number << ":" << column << "\n";
   } else {
     if (frame->IsConstructor()) {
-      fprintf(fp, "%s [constructor] (%s:%i:%i)\n", *fn_name_s, *script_name, line_number, column);
+      out  << *fn_name_s << " [constructor] (" << *script_name << ":" << line_number << ":" << column << ")\n";
```

Comment:
Another long line, overall I think we need to go through an make sure we've not pushed lines over the limit, especially since we may turn on the linter soon.